lib/panoptes/client/subjects.rb

```diff
@@ -16,6 +16,19 @@ def subjects(subject_set_id: nil, workflow_id: nil)
         response.fetch("subjects")
       end
 
+      # Determine whether a particular subject is accessible to a particular project
+      #
+      # @param subject_id [Integer]
+      # @param project_id [Integer]
+      # @return nil or the subject
+      def subject_in_project?(subject_id, project_id)
```

Comment:
Would it be better to do this as a more generic method, like `def subject(subject_id, project_id: nil)` which takes an optional project_id to scope it, and returns the subject itself. This might be usable in more contexts, and for Caesar in this case the API call would just become `subject(123, project_id: 456).present?`.

Reply:
This is a good idea.